cmd/webhook/main.go

```diff
@@ -29,22 +35,40 @@ var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	v1alpha1.SchemeGroupVersion.WithKind("Stack"):                &expv1alpha1.Stack{},
 }
 
+var (
+	storageClassLister v1.StorageClassLister
+)
+
 func main() {
+	restConfig := sharedmain.ParseAndGetConfigOrDie()
+
+	k8sClient, err := kubernetes.NewForConfig(restConfig)
```

Comment:
This is bizarre but, could we utilize knative pkg's injection here so that we have just one way of setting up informers?